extensions/wikia/AdEngine/js/GptSraHelper.js

```diff
@@ -0,0 +1,30 @@
+/*global define,setTimeout*/
+/*jshint maxlen:125, camelcase:false, maxdepth:7*/
+define('ext.wikia.adEngine.gptSraHelper', [
+	'wikia.log'
+], function (log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.gptSraHelper',
+		flushed = false,
+		sraSlots = [
+			'CORP_TOP_LEADERBOARD',
+			'HOME_TOP_LEADERBOARD',
+			'HUB_TOP_LEADERBOARD',
+			'INVISIBLE_SKIN',
+			'TOP_LEADERBOARD'
+		];
+
+	function shouldFlush(slotName) {
+		if (sraSlots.indexOf(slotName) === -1) {
```

Comment:
Please add some logging. This can become tricky at one point, since the flushed is per-module var, so it won't work if multiple GPT-based providers would like to use SRA.